specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -1368,6 +1412,44 @@
         }
       }
     },
+    "NotebookListCredentialsResult": {
+      "type": "object",
+      "properties": {
+        "primaryAccessKey": {
+          "type": "string"
+        },
+        "secondaryAccessKey": {
+          "type": "string"
+        }
+      }
+    },
+    "NotebookResourceInfo": {
+      "type": "object",
+      "properties": {
+        "fqdn": {
+          "type": "string"
+        },
+        "resourceId": {
+          "type": "string"
+        },
```

Comment:
what resource is this pointing to?  I thought you don't have a resource representation of the notebook

Reply:
You mean NotebookResourceInfo.resourceId? This is some dataplane resourceId that notebook needs to initialize component. If you look at the example in prepareNotebook, it's not an ARM Id.

Reply:
I would use a different property name here.  "resourceId" is going to mean an ARM resource ID by default to most users.  At the very least this needs a description.

Reply:
Notebook RP returns us this property with name ResourceId, so I'm keeping the name to be consistent with Notebook, but I will add description to make it less confusing